WorkWithFiles/work_files.rb

```diff
@@ -30,7 +20,7 @@ def create (file_name, access_right = 'w') #w, w+, a, a+
         rescue ArgumentError => e
           puts "Got the following error(Argument): '#{e}'"
           puts 'Error to create file'
```

Comment:
please, combine this two msg into one like:
 puts "Failed to create '#{file_name}'  file with the following error: '#{e}' "